### DISCUSSION TITLE
OpenXLA Community Meeting Agenda: 10/18 @ 8AM PDT

### DISCUSSION THREAD

**Discussion by theadactyl** [openxla/community#25] [Announcements]
Here's our agenda for the Oct. 18 meeting at 8am PT ([Zoom Link](https://us02web.zoom.us/j/87557882524?pwd=QUJZQlZub0tRTk1CbCt4eFYzZ0lJUT09) | [Meeting Folder](https://github.com/openxla/community/tree/main/meetings/20221018-openxla-community-meeting)): 

[5 min] Welcomes & housekeeping 
[30-45 min] Technical presentations: 
  * StableHLO compatibility (Kevin Gleason, Google)
  * GML_ST (Frederik Gossen, Google) 
  * Cuda Graph RFC (Eugene Zhulenev, Google)
  
[5 min] Updated timeline for XLA migration
[5 min] Open Discussion

Any topics not listed here you wanted to discuss? Let us know in the comments.

**Comment by theadactyl** [comment]
Update: 2 technical presentations have slides available for review in the meeting directory: https://github.com/openxla/community/tree/main/meetings/20221018-openxla-community-meeting

**Comment by joker-eph** [comment]
Here is the recording for the talks: https://youtu.be/7X_-Ji2LuEk
